src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/AdjudicationMigrateDto.kt

```diff
@@ -100,6 +103,16 @@ data class MigrateDamage(
 
 )
 
+@Schema(description = "evidence related to adjudication")
+data class MigrateEvidence(
+  @Schema(description = "evidence code")
+  val evidenceCode: EvidenceCode,
+  @Schema(description = "evidence details / statement")
+  val details: String,
+  @Schema(description = "this is the reporter, we could use the investigator in this field, or createdBy user")
+  val reporter: String,
```

Comment:
in our system, we need a reporter, who owns this evidence.  so in theory, we could take the investigator username as this entry?